docs/common-problems.asciidoc

```diff
@@ -47,7 +47,9 @@ the {apm-agents-ref}/index.html[agent documentation] for details on what is auto
 === HTTP 400: Data decoding error / Data validation error
 
 The most likely cause for this is that you are using incompatible versions of agent and APM Server.
-For instance, APM Server 6.2.0 changed the Intake API spec and requires a minimum version of each agent.
+For instance, APM Server 6.5 redesigned the Intake API and requires a minimum version of each agent.
```

Comment:
Hmm, this is not true because we still keep v1 for backwards compatibility, right? (the other way around is true: latest agent versions require 6.5)
The change we made in 6.2 was more disruptive ;)